<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>123</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>418</width>
    <height>123</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form {border:1px solid rgb(0, 0, 0); }</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>403</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Pendiente: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="pendienteTxt">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="calibrarChk">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Calibrar</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="ordenadaTxt">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Unidad:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="unidadTxt">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Ordenada: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nombre: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nombreTxt">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="activadoChk">
      <property name="text">
       <string>Activar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="tipoInput">
      <property name="text">
       <string>Input Analogico</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calibrarChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>pendienteTxt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activadoChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>calibrarChk</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrarChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>ordenadaTxt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activadoChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>nombreTxt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activadoChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>unidadTxt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
